src/utils/logging.py

```diff
@@ -75,10 +63,11 @@ def setup_logging() -> None:
     debug_all = os.environ.get("DEBUG_ALL", "false").lower() == "true"
     log_level = os.environ.get("LOG_LEVEL", "INFO").upper()
 
-    # Root logger level - set to DEBUG to allow all loggers to emit messages
+    # Root logger level - WARNING by default, DEBUG only if DEBUG_ALL is set
+    root_level = "DEBUG" if debug_all else "WARNING"
```

Comment:
So what is the setting for DEBUG level on x2a-convertor code and WARING on third-party libraries?